themes/theme-core/tailwind.config.ts

```diff
@@ -5,6 +5,7 @@ import { preset } from './src/preset';
 // Figma File: https://www.figma.com/file/RiWJBV4Z8L8ycVvUuMYXbm/%F0%9F%93%93-CR---Components-2022?node-id=1452-1785&t=YaLGVHzniD5mOtbJ-0
 
 export default {
+  plugins: [require('tailwindcss-react-aria-components')],
```

Comment:
Can't we add it to the preset? :)

Reply:
Here:

https://github.com/marigold-ui/marigold/blob/main/packages/theme-preset/src/create-preset.ts#L17